Create uno.md (#49)
ScootKit/scnx-docs: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/fun/uno.md

# Uno

> Lasst eure User gegeneinander Uno spielen!

Reviewer: ```suggestion
Lasse deine Nutzer gegeneinander Uno spielen!
```

Author: done

Reviewer: No?

Author: now jet

> Lasse deine User gegeneinander Uno spielen!

Reviewer: ```suggestion
Lasse deine Nutzer gegeneinander Uno spielen!
```

Author: no issue in my code

<ModuleOverview moduleName="uno" />

## Funktionen {#features}

> Uno ist ein klassisches Kartenspiel - jetzt auch für deinen eigenen Bot verfügbar! Wenn du mehr über das Spiel erfahren möchten, schau dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.

Reviewer: ```suggestion
Uno ist ein klassisches Kartenspiel - jetzt auch für deinen eigenen Bot verfügbar! Wenn du mehr über das Spiel erfahren möchtest, schau dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.
```

Author: solved

> Uno ist ein klassisches Kartenspiel - jetzt auch für deinen eigenen Bot verfügbar! Wenn du mehr über das Spiel selbst erfahren möchten, schau dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.

Reviewer: ```suggestion
Uno ist ein klassisches Kartenspiel - jetzt auch für deinen eigenen Bot verfügbar! Wenn du mehr über das Spiel erfahren möchtest, schau dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.
```

Author: solved

> Uno ist ein klassisches Kartenspiel - jetzt auch für deinen Eigenen Bot verfügbar! Wenn du mehr über das Spiel selbst erfahren möchten, schauen dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.

Reviewer: ```suggestion
Uno ist ein klassisches Kartenspiel - jetzt auch für deinen eigenen Bot verfügbar! Wenn du mehr über das Spiel selbst erfahren möchten, schau dir den [Wikipedia Artikel](https://de.wikipedia.org/wiki/Uno_(Kartenspiel)) an.
```

Author: done

Author: no isue

Jeder Spieler beginnt mit sieben Karten. Die anderen Spieler können nur die Anzahl der Karten jedes Spielers sehen. Wenn ein Spieler keine Karte spielen kann, muss er eine Karte ziehen. Wenn ein Spieler keine Karten mehr hat, gewinnt er das Spiel.

## Einrichtung {#setup}

> Die Einrichtung ist wirklich einfach – aktivieren sie einfach das Modul in [deinem SCNX-Dashboard](https://scnx.app/de/glink?page=bot/modules?query=uno&ref=scnx-app-docs) und schon kann es losgehen!

Reviewer: ```suggestion
Die Einrichtung ist wirklich einfach - aktiviere einfach das Modul in [deinem SCNX-Dashboard](https://scnx.app/de/glink?page=bot/modules?query=uno&ref=scnx-app-docs) und schon kann es losgehen!
```

Author: done

Author: no issue

## Verwendung {#usage}

> Um ein Uno-Spiel zu starten, führe den Slash-Befehl `/uno` aus - andere Spieler können dann über den Knopf „Spiel beitreten“ beitreten. Nach 90 Sekunden oder nachdem der Benutzer, der den Befehl ausführt, auf den Knopf „Spiel sofort starten“ geklickt hat, beginnt das Spiel.

Reviewer: ```suggestion
Um ein Uno-Spiel zu starten, führe den Slash-Befehl `/uno` aus - andere Spieler können dann über den Knopf „Spiel beitreten“ beitreten. Nach 90 Sekunden oder nachdem der Benutzer, der den Befehl ausgeführt hat, auf den Knopf „Spiel sofort starten“ geklickt hat, beginnt das Spiel.
```

Author: solved

> Um ein Uno-Spiel zu starten, führen sie den Slash-Befehl `/uno` aus – andere Spieler können dann über die Schaltfläche „Spiel beitreten“ beitreten. Nach 90 Sekunden oder nachdem der Benutzer, der den Befehl ausführt, auf die Schaltfläche „Spiel starten“ geklickt hat, beginnt das Spiel.

Reviewer: ```suggestion
Um ein Uno-Spiel zu starten, führe den Slash-Befehl `/uno` aus - andere Spieler können dann über den Knopf „Spiel beitreten“ beitreten. Nach 90 Sekunden oder nachdem der Benutzer, der den Befehl ausführt, auf den Knopf „Spiel sofort starten“ geklickt hat, beginnt das Spiel.
```

Author: done

Author: no issue

> :::tip Spiel Thread 
> Es ist am besten, einen Thread für Spieldiskussionen zu erstellen, damit der Kanal nicht zu voll mit spielbezogenen Nachrichten wird.
> :::

Reviewer: ```suggestion
:::tip Spiel-Thread 
Es ist am besten, einen Thread für Spieldiskussionen zu erstellen, damit der Kanal nicht zu voll mit spielbezogenen Nachrichten wird.
:::
```

Author: done

Author: no issue

> Inaktive Spieler werden nach 60 Sekunden, wenn sie an der Reihe sind, verwarnt und nach weiteren 60 Sekunden aus dem Spiel geworfen.

Reviewer: ```suggestion
Inaktive Spieler werden 60 Sekunden, nachdem sie an die Reihe gekommen sind verwarnt und nach weiteren 60 Sekunden aus dem Spiel entfernt.
```

Author: done

Author: no issue

Das Spiel selbst hat keine zeitliche Begrenzung.

### Wie man spielt {#how-to-play}
Spieler können nur Karten spielen, die dem aktuellen Wert oder der aktuellen Farbe der zuletzt gespielten Karte entsprechen. Ausnahmen gelten für die Sonderkarten „Farbwahl“.

> In den meisten Fällen verwendet jeder Spieler eine Karte, indem er eine Karte aus seinem Kartenstapel auswählt, wonach der nächste Spieler an der Reihe ist - beachte, dass die Zugreihenfolge mithilfe der Sonderkarte „Umkehren“ umgekehrt werden kann.

Author: done

> In den meisten Fällen verwendet jeder Spieler eine Karte, indem er eine Karte aus seinem Kartenstapel auswählt, wonach der nächste Spieler an der Reihe ist - beachten Sie, dass die Zugreihenfolge mithilfe der Sonderkarte „Umkehren“ umgekehrt werden kann.

Reviewer: ```suggestion
In den meisten Fällen verwendet jeder Spieler eine Karte, indem er eine Karte aus seinem Kartenstapel auswählt, wonach der nächste Spieler an der Reihe ist - beachte, dass die Zugreihenfolge mithilfe der Sonderkarte „Umkehren“ umgekehrt werden kann.
```

Author: solved

> In den meisten Fällen verwendet jeder Spieler eine Karte, indem er eine Karte aus seinem Kartenstapel auswählt, und dann ist der nächste Spieler an der Reihe – beachten Sie, dass die Zugreihenfolge mithilfe der Sonderkarte „Umkehren“ umgekehrt werden kann.

Reviewer: ```suggestion
In den meisten Fällen verwendet jeder Spieler eine Karte, indem er eine Karte aus seinem Kartenstapel auswählt, wonach der nächste Spieler an der Reihe ist - beachte, dass die Zugreihenfolge mithilfe der Sonderkarte „Umkehren“ umgekehrt werden kann.
```

Author: done

> ### Karten und Specialeffekte {#cards}

Reviewer: ```suggestion
### Karten und Spezialeffekte {#cards}
```

Author: correckted

> Es gibt normale Zahlenkarten von 0-9 in den Farben Blau, Grün, Rot und Gelb sowie Sonderkarten. Zu den Sonderkarten gehört die „Farbwahl“, mit der eine gewünschte Farbe für die nächste Karte ausgewählt werden kann; „Umkehren“, was die Spielrichtung ändert; und „Überspringen“, wodurch der nächste Spieler übersprungen wird.

Author: done

> Es gibt normale Zahlenkarten von 0-9 in den Farben Blau, Grün, Rot und Gelb sowie Sonderkarten. Zu den Sonderkarten gehört die Funktion „Farbwahl“, mit der eine gewünschte Farbe für die nächste Karte ausgewählt werden kann; „Reverse“, was die Spielrichtung ändert; und „Überspringen“, wodurch der nächste Spieler übersprungen wird.

Reviewer: ```suggestion
Es gibt normale Zahlenkarten von 0-9 in den Farben Blau, Grün, Rot und Gelb sowie Sonderkarten. Zu den Sonderkarten gehört die „Farbwahl“, mit der eine gewünschte Farbe für die nächste Karte ausgewählt werden kann; „Reverse“, was die Spielrichtung ändert; und „Überspringen“, wodurch der nächste Spieler übersprungen wird.
```

Author: done

Außerdem gibt es noch die Karten „Ziehe 2“ und „Farbwahl und ziehe 4“: Dadurch wird der nächste Spieler gezwungen, zwei oder vier Karten zu ziehen, sofern er nicht auch eine dieser Karten in seinem Kartenstapel hat. Hat er eine solche Karte, kann er damit die Anzahl der zu ziehenden Karten erhöhen und an den nächsten Spieler weitergeben, wo sich der ganze Vorgang wiederholt.

> ## Commands {#commands}

Reviewer: ```suggestion
## Befehle {#commands}
```

Author: sorry i miss it

<SlashCommandExplanation />

> | Command | Description                                                                                                            |
> |---------|------------------------------------------------------------------------------------------------------------------------|
> | `/uno`  | Startet ein Uno-Spiel im aktuellen Kanal. Weitere Informationen zur Funktionsweise finden sie im [Abschnitt zur Verwendung](#usage) oben. 

Reviewer: ```suggestion
| Befehl | Beschreibung                                                                                                            |
|---------|------------------------------------------------------------------------------------------------------------------------|
| `/uno`  | Startet ein Uno-Spiel im aktuellen Kanal. Weitere Informationen zur Funktionsweise findest du im [Abschnitt zur Verwendung](#usage) oben. 
```

Author: sorry hatte ich übersehen 

aber done

                                                                                                                                   |

## Fehlerbehebung {#troubleshooting}

<details>

>     <summary>Wie kann ich eine Karte spielen/Wo ist mein Kartendeck?</summary>

Reviewer: ```suggestion
    <summary>Wie kann ich eine Karte spielen / Wo ist mein Kartendeck?</summary>
```

Author: solved

>     <li>Du kannst eine neue Nachricht mit deinem aktuellen Kartendeck erhalten, einschließlich einer Schaltfläche zum Aktualisieren, indem du auf die Schaltfläche „Deck anzeigen“ in der ursprünglichen Spielnachricht klickst.</li>

Reviewer: ```suggestion
    <li>Du kannst eine neue Nachricht mit deinem aktuellen Kartendeck und einem Knopf zum Aktualisieren erhalten, indem du auf den Knopf „Eigene Karten ansehen“ in der ursprünglichen Spielnachricht klickst.</li>
```

Author: solved

>     <li>Sie können eine neue Nachricht mit Ihrem aktuellen Kartendeck erhalten, einschließlich einer Schaltfläche zum Aktualisieren, indem du auf die Schaltfläche „Deck anzeigen“ in der ursprünglichen Spielnachricht klicken.</li>

Reviewer: ```suggestion
    <li>Du kannst eine neue Nachricht mit deinem aktuellen Kartendeck erhalten, einschließlich einem Knopf zum Aktualisieren, indem du auf die Schaltfläche „Deck anzeigen“ in der ursprünglichen Spielnachricht klickst.</li>
```

Author: done

</details>
<details>

>     <summary>Der Bot sagt: "⚠️️ Du musst die Uno!-Button nutzen, bevor du die vorletzte Karte legst!" - aber wieso?</summary>
>     <li>u musst zuerst auf "Uno!"-Knopf der ursprünglichen Spielnachricht klicken, bevor du deine vorletzte Karte ausspielst. Das gilt nicht für das Ausspielen der letzten Karte.</li>

Reviewer: ```suggestion
    <summary>Der Bot sagt: "⚠️️️ Du musst den Uno!-Button nutzen, bevor du die vorletzte Karte legst!" - wieso?</summary>
    <li>Du musst zuerst auf den "Uno!"-Knopf der ursprünglichen Spielnachricht klicken, bevor du deine vorletzte Karte ausspielst. Das gilt nicht für das Ausspielen der letzten Karte.</li>
```

Author: i don't know wath happend during my las correction, but i fixed it now

>     <summary>TDer Bot sagt: "⚠️️ Sie müssen die Uno!-Taste verwenden, bevor Sie Ihre vorletzte Karte verwenden!" - aber warum?</summary>
>     <li>Sie müssen zuerst auf Uno! klicken. Klicken Sie auf die Schaltfläche auf der ursprünglichen Spielnachricht, bevor Sie Ihre vorletzte Karte ausspielen. Dies gilt nicht für das Ausspielen der letzten Karte.</li>

Reviewer: ```suggestion
    <summary>Der Bot sagt: "⚠️️️ Du musst den Uno!-Button nutzen, bevor du die vorletzte Karte legst! " - wieso?</summary>
    <li>Du musst zuerst auf "Uno!"-Knopf der ursprünglichen Spielnachricht klicken, bevor du deine vorletzte Karte ausspielst. Das gilt nicht für das Ausspielen der letzten Karte.</li>
```

Author: done
